coordinates/sites.json

```diff
@@ -75,12 +75,18 @@
         "name": "Roque de los Muchachos, La Palma",
         "longitude_unit": "degree",
         "latitude_unit": "degree",
-        "latitude": 28.758333333333333,
+        "latitude": 28.758333333333,
         "elevation_unit": "meter",
-        "longitude": 342.12,
+        "longitude": -17.88,
         "timezone": "Atlantic/Canary",
         "aliases": [
-            "Roque de los Muchachos"
+            "Roque de los Muchachos",
+            "Isaac Newton group",
+            "ING",
```

Comment:
Note that I think it may be problematic to specify the aliases here as being all the different telescopes.  the "la palma" location is presumably some generic spot on the mountain, *not* the specific observatory locations.  So I'd think it's better to drop the telescope names unless there's a more refined location (in which case they would get separate entries)

Reply:
The La Palma location is, according to Google Maps, a spot on the edge of the caldera not corresponding to any actual telescope. For my project, I do not need the location at the precision of individual telescopes, so added some aliases for telescopes under the observatory headings. That may of course not be sensible for other use cases.

Reply:
@rwesson - I agree with @eteq on this one, we should either list the precise/official location of the telescopes/individual observatories or just have the generic ORM one without precise aliasing.